addons/website/static/src/js/content/inject_dom.js

```diff
@@ -15,6 +15,25 @@ export function unhideConditionalElements() {
     document.head.appendChild(styleEl);
     const conditionalEls = document.querySelectorAll('[data-visibility="conditional"]');
     for (const conditionalEl of conditionalEls) {
+        // In the case of a mega menu block, hide the mega menu link in the bar according to the same rule
+        // Change the visibility for Desktop and Mobile views
+        if (conditionalEl.classList.contains("s_mega_menu_odoo_menu")) {
+            const megaMenuDesktop = conditionalEl.closest("li[role~='presentation']");
+            const menusDesktop = megaMenuDesktop.parentElement.querySelectorAll('li');
```

Comment:
I don't think you want all descendants li but all **children** ones :pray:

Reply:
I haven't found a way to get the index on the index of the element when replacing "querySelectorAll" with alternatives to get the children